atlas-slurm

```diff
@@ -98,9 +98,15 @@ while getopts ":c:w:m:p:j:g:l:o:e:n:f:q:s:r:v:u:" o; do
     esac
 done
 
-if [[ ! -n "${workingDir}" ]]; then
-    workingDir="$scriptDir"
-elif [[ ! -d "${workingDir}" ]]; then
+if [ -z "$commandString" ]; then
+    die "Need at least a command string for SLURM submission"
+fi
+
+if [ -z "$workingDir" ]; then
```

Comment:
Perhaps an `elif` instead of a new `if` block, even though by virtue of sequence the result is the same.

LGTM.

Reply:
Can do